src/GafferCyclesModule/GafferCyclesModule.cpp

```diff
@@ -466,10 +467,12 @@ BOOST_PYTHON_MODULE( _GafferCycles )
 #else
 	py::scope().attr( "withTextureCache" ) = false;
 #endif
-	if( ccl::openimagedenoise_supported() )
-		py::scope().attr( "hasOpenImageDenoise" ) = true;
-	else
-		py::scope().attr( "hasOpenImageDenoise" ) = false;
+	py::scope().attr( "hasOpenImageDenoise" ) = ccl::openimagedenoise_supported();
+#ifdef WITH_CYCLES_PATH_GUIDING
+	py::scope().attr( "hasPathGuiding" ) = true;
+#else
+	py::scope().attr( "hasPathGuiding" ) = false;
+#endif
```

Comment:
Since we're in control of the builds, and will be building with path guiding on, do we really need this?

Reply:
Yep lets get rid of it